server/daos/booking_dao.js

```diff
@@ -113,4 +113,22 @@ exports.deleteBookingTable = function() {
             else resolve(null);
         });
     })
+}
+
+
+//it allows you to delete a booking
+exports.deleteBooking = function({lecture_id,student_id}) {
+    return new Promise ((resolve,reject) =>{
+        const sql = 'DELETE FROM Bookings WHERE lecture_id= ? AND student_id= ?'
```

Comment:
I'm pretty sure that we should not actually `DELETE` a booking, but just update the `deleted_at` field. In this way we are able to retrieve the cancelled bookings in the anaysis.

As a memo: when a normal `GET` of the bookings is performed we should pay attention to that field

Reply:
I noticed that field is not used in the dao, I'm fixing it